README.md

```diff
@@ -404,18 +404,25 @@ You can read about [getting started](https://julialang.org/manual/getting-starte
 
 The following distributions include julia, but the versions may be out of date due to rapid development:
 
+<a href="https://repology.org/metapackage/julia">
+    <img src="https://repology.org/badge/vertical-allrepos/julia.svg" alt="Packaging status" align="right">
```

Comment:
I'm fine with linking to the summary page but the image would be a bit misleading then.